client/app/filters/datetime.js

```diff
@@ -5,7 +5,7 @@ export default function init(ngModule) {
 
   ngModule.filter('dateTime', clientConfig =>
     function dateTime(value) {
-      if (!value) {
+      if (!value || !value.isValid()) {
```

Comment:
I noticed that we call `moment` on `value` below, which implies that we might send a string to this function. Maybe check `isValid` after the conversion to a `moment` object?